oven

```diff
@@ -222,6 +222,10 @@ function mount_image() {
     mount --bind /dev "${opt_mount_point}/dev"
     mount --bind /sys "${opt_mount_point}/sys"
     mount -t proc none "${opt_mount_point}/proc"
+    
+    # Mount binfmt_misc seems to be needed for ubuntu 16.04+ at least
+    log "INFO" "Mouting binfmt_misc"
+    mount binfmt_misc -t binfmt_misc /proc/sys/fs/binfmt_misc  
```

Comment:
I think we should check first if binfmt_misc is already mounted. On environments where binfmt_misc is mounted, this line fails (`mount: binfmt_misc is already mounted`).

Reply:
Ok, sounds good. I'll revise this shortly.